### DISCUSSION TITLE
Lesson 7 - My first 2 test Revert

### DISCUSSION THREAD

**Discussion by Loupsteb** [Cyfrin/foundry-full-course-cu#1312] [Q&A] [answered]
Hi, i'm at the begining of the course and i try to reproduce what Patrick did in his course but i've some trouble.
I try to make the 2 first Patricks test but its revert and when i try to log its not working maybe i can have some help.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/52604984/66ba55d0-3af7-48d7-b03c-dfc773eff9ba)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/52604984/6b1712bb-df25-4709-a38d-9f5316bbd52a)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/52604984/88411e6a-b0c8-4067-ae39-729833af112a)

**Comment by vascofcd** [comment, marked as answer]
Try `function setUp() external {...` instead of `function testSetUp() external {...`
